lib/views/introduction/slides/battery_optimization.dart

```diff
@@ -65,7 +65,7 @@ class _IntroductionBatteryOptimizationSlideState
     }
 
     return IntroductionBaseSlide(
-      imagePath: "assets/battery.png",
+      imagePath: "battery.png", //@TODO replace image
```

Comment:
```suggestion
      imagePath: "battery.png", // TODO(nucleus): replace image
```

Please use the Flutter-style TODO's. They will also show up better in editors like VSCode with the Flutter plugin.